### PR TITLE
T108559 fix admin lock

### PR DESCRIPTION
**Pull request by joaopampanini**

### CODE REVIEW

File: admin_page_lock/handlers.py [hunk 1]
```diff
@@ -34,26 +31,28 @@ def _get_lock_settings(self):
         # settings somehow or defined and stored in database. Right now, the
         # settings is same for all pages!
         lock_settings = {
-            'csrf_token': get_token(self.page_settings['req']) if not DISABLE_CRSF_TOKEN else '',  # noqa: E501
-            'homepage': HOMEPAGE,
-            'messages': self.page_settings['messages'],
-            'user_reference': self.page_settings['user_reference'],
+            "csrf_token": get_token(self.page_settings["req"])
+            if not DISABLE_CRSF_TOKEN
+            else "",  # noqa: E501
```
Comment (geckon) [inline admin_page_lock/handlers.py:36]: The comment is not needed here.

File: admin_page_lock/mixins/base_mixin.py [hunk 2]
```diff
@@ -21,13 +22,14 @@ class BaseLockingMixin(object):
...
+            "page_lock_template_data": json.dumps(data),
+            "page_lock_api_interval": int(API_INTERVAL),  # must be integer
         }
 
         if (
-            data['is_locked'] and
-            data['page_lock_settings']['user_reference'] != data['locked_by']  # noqa: E501
+            data["is_locked"]
+            and data["page_lock_settings"]["user_reference"]
+            != data["locked_by"]  # noqa: E501
```
Comment (geckon) [inline admin_page_lock/mixins/base_mixin.py:32]: Comment also not needed here.

**Review by ride90 — APPROVED** [review]

File: admin_page_lock/models/database_model.py [hunk 3]
```diff
@@ -28,18 +27,21 @@ class DatabasePageLockModel(BasePageLockModel, models.Model):
...
-            'active': True,
-            'url': page_settings['page_url'],
+            "active": True,
+            "url": page_settings["page_url"],
         }
         if not URL_IGNORE_PARAMETERS:
-            query_kwargs['url_parameters'] = page_settings['page_url_parameters']  # noqa: E501
+            query_kwargs["url_parameters"] = page_settings[
+                "page_url_parameters"
+            ]  # noqa: E501
```
Comment (geckon) [inline admin_page_lock/models/database_model.py:44]: dtto

File: admin_page_lock/settings.py [hunk 4]
```diff
@@ -5,82 +5,88 @@
...
 HOMEPAGE = getattr(settings, HOMEPAGE_REFERENCE, HOMEPAGE_DEFAULT)
 
 # Keep DB locks (possible to see history).
 KEEP_DB_LOCKS_DEFAULT = False
-KEEP_DB_LOCKS_REFERENCE = 'PAGE_LOCK_KEEP_DB_LOCKS'
-KEEP_DB_LOCKS = getattr(settings, KEEP_DB_LOCKS_REFERENCE, KEEP_DB_LOCKS_DEFAULT)  # noqa: E501
+KEEP_DB_LOCKS_REFERENCE = "PAGE_LOCK_KEEP_DB_LOCKS"
+KEEP_DB_LOCKS = getattr(
+    settings, KEEP_DB_LOCKS_REFERENCE, KEEP_DB_LOCKS_DEFAULT
+)  # noqa: E501
```
Comment (geckon) [inline admin_page_lock/settings.py:50]: dtto

File: admin_page_lock/settings.py [hunk 5]
```diff
@@ -5,82 +5,88 @@
...
-    'message_refresh': _('Click "REFRESH" button to start editting.'),
-    'message_problem': _('Something happened, you will be redirected to homepage.'),  # noqa: E501
+    "message_locked": _("Page is locked."),
+    "message_reload": _(
+        'You are the one who can edit. Click "RELOAD" button to stay on the page.'
+    ),  # noqa: E501
+    "message_refresh": _('Click "REFRESH" button to start editting.'),
+    "message_problem": _(
+        "Something happened, you will be redirected to homepage."
+    ),  # noqa: E501
```
Comment (geckon) [inline admin_page_lock/settings.py:61]: dtto

**Review by geckon — APPROVED** [review]
Approving but with comments, please remove the unnecessary `noqa` comments before merging.

**Pull request merged**